src/sp/it/pl/util/functional/Util.java

```diff
@@ -286,7 +286,7 @@ static void repeat(int n, IntConsumer action) {
 	 * @throws NullPointerException if argument is null
 	 */
 	static <E, C extends Comparable<? super C>> Comparator<E> by(Function<? super E,? extends C> extractor) {
-		noNull(extractor);
+		findNonNull(extractor);
```

Comment:
`findNonNull` is not a replacement as it does not throw an exception (it is written in the javadoc that it can return null). Use `noNull`. This error is spread through the whole file, pls make sure its correct in the whole project.

Reply:
I think that was just a fail while I did search & replace